### DISCUSSION TITLE
Option to make hidden function the same way as it did prior to late 2015

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#15330] [Ideas]
Before October 2015, [hidden used to not fade out slider bodies and just faded out the numbers of sliders](https://www.youtube.com/watch?v=jM9_PGQXQFY), but then there was a significant update that changed that and now slider bodies fade out with hidden. I have no issue with this, but with all of the mod customization settings making it into the game (and specifically one for hidden), I was wondering if there could be a customization setting for hidden that makes it so that it functions the same way as it did prior to late 2015. I think that it would be pretty nostalgic for me and a lot of other players who have been playing the game since then.

**Comment by peppy** [comment]
Have you tried the newly added option? It's quite close to what you're after, apart from hitcircles still fading.

I'm not against adding more settings, but I don't think we should necessarily be adding any and every setting under the sun. If there's enough support for this then it can definitely be considered.

**Reply by Flamiii** [reply]
I've tried the new option and I definitely get where you're coming from since with this option enabled the slider bodies don't fade out, but the hitcircles and the numbers of the sliders don't fade out either which makes it feel less like the old hidden and more like nomod but without any approach circles.

I also understand your concern regarding adding too many unnecessary options, but I feel that for this case in particular, given that a large portion of the player base has been playing since 2015 or earlier, I think that it would be warranted since they would probably recognize why a setting like this would be wanted in the game.

**Reply by smoogipoo** [reply]
There are more players post-2015 than pre-2015. Just as a personal opinion (thus not necessarily blocking this from ever happening) as a 2013 OWC player myself, I don't think people would care about such a setting, it would be purely for nostalgia's sake and almost guaranteed to be a one-time use as it's pretty much inferior in 99% of cases.

**Reply by Flamiii** [reply]
What you're saying is definitely true, but at the same time I think that pretty much all of the 'fun' mods kind of fit into that category of one-time use mods that are initially fun but end up not being used that much if at all in actual gameplay. I do get what you're saying though; obviously a feature like this would be pretty low priority at the moment, so perhaps it's something that could potentially be revisited in the future.
